crates/bevy_ecs/src/schedule_v3/executor/multi_threaded.rs

```diff
@@ -243,10 +249,12 @@ impl MultiThreadedExecutor {
 
         for system_index in ready_systems.ones() {
```

Comment:
The actual bitset used here is `ready_systems` not `self.ready_systems`.

Reply:
We're calling `&mut self` methods in the loop, so we can't hold onto `&self.ready_systems`. So I copy `self.ready_systems` into an empty bitset and iterate that one instead.

Reply:
Ah I should have been clearer, the safety comment should just be updated to reflect that the parameter `ready_systems` is the subject of the safety invariant, not `self.ready_systems` which is being mutated in this function. From the perspective of this function, it shouldn't matter what `self.ready_systems`, only what's passed in via the `ready_systems` function parameter.

Reply:
Looks good to me now.